api/data_refinery_api/exceptions.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.exceptions import APIException
+from rest_framework.views import exception_handler
+
+
+def custom_exception_handler(exc, context):
+    response = exception_handler(exc, context)
+
+    if response:
+        if isinstance(exc, APIException):
+            response.data["error_code"] = exc.detail.code
```

Comment:
What's this doing? Is this an HTTP response code?

Reply:
It's different than the response code number like `400`/`403`. Maybe `type` would be a better field name. 
I think it would be good to return some kind of identifier for the type of error so that, if we ever add more custom 400 exceptions, the client could differentiate between them.

Reply:
Ah ok. Yeah I'm cool with adding an `error_type` field as long as its human readable.